pyincore/analyses/socialvulnerabilityscore/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 University of Illinois and others. All rights reserved.
```

Comment:
This should match the original copyright year, 2021

Reply:
I have already updated. Maybe you're using the old one? Can you pull one more time?

Reply:
oh my bad, changed it.